training/src/main/scala/com/airbnb/aerosolve/training/pipeline/ResultUtil.scala

```diff
@@ -0,0 +1,19 @@
+package com.airbnb.aerosolve.training.pipeline
+
+import com.airbnb.aerosolve.training._
+
+/*
+ * Writes to result file
+ */
+object ResultUtil {
+  def writeResults(metrics: Array[(String, Double)], resultOutputPath: String) = {
+    // Put into a one line json format
+    val elements = metrics.map{ x =>
+      val metricStr = if (x._1 contains "THRESHOLD") x._1 else x.toString
```

Comment:
I added comments for the output format, but I wasn't sure what to test in a unit test